docs/UserGuide.md

```diff
@@ -1,226 +1,23 @@
----
-layout: page
-title: User Guide
----
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
```

Comment:
The existing user guide seems to have been deleted in your copy, could you please try adding your changes within the existing user guide?